e2e/save_report.js

```diff
@@ -56,6 +56,7 @@ const saveReport = async () => {
         DIAGNOSTIC_TEAM_ID,
         FAILURE_MESSAGE,
         TM4J_ENABLE,
+        TM4J_CYCLE_KEY,
```

Comment:
Thanks @saturninoabril 
We are till using ```TM4J``` name ? I thought it has changed !

Reply:
0/5 on name to use. Zephyr sounds unfamiliar to me :)